src/docs/training-and-learning/training-from-the-platform-services-team.md

```diff
@@ -24,7 +24,7 @@ Learning how to build in OpenShift can be hard. We’ve put together a number of
 
 ## OpenShift 101 and 201
 
-The BC Gov Provate Cloud PaaS OpenShift 101 and 201 courses include a theory 'workshop' component, and a practical series of 'lab exercises'. OpenShift 101 is recommended for all new teams joining the platform. OpenShift 201 is a more advanced course for platform users who want to advance their skills in OpenShift.
+The BC Gov Private Cloud PaaS OpenShift 101 and 201 courses include a theory 'workshop' component, and a practical series of 'lab exercises'. OpenShift 101 is recomended for all new teams joining the platform. OpenShift 201 is a more advanced course for platform users who want to advance their skills in OpenShift.
```

Comment:
I would add B.C. Gov going forward based on the correct use of Acronyms 
https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide/writing-guide/abbreviations?keyword=acronyms#province

Reply:
right! I'm just fixing the typo where I see them, but there might be more. I'll find a time to correct all the `BC Gov` to `B.C. Gov` and fix all the typos together so I don't miss any!